libs/blocks/section-metadata/section-metadata.js

```diff
@@ -44,6 +44,12 @@ function handleLayout(text, section) {
   section.classList.add(layoutClass);
 }
 
+function handleDelay(time, section) {
+  if (!(time || section)) return;
+  section.classList.add('hide-sticky-section');
+  setTimeout(() => { section.classList.remove('hide-sticky-section'); }, time);
+}
```

Comment:
Multiples of 1000 to get seconds is more of developer thinking. I think most people would put `delay | 5` and expect a five second delay.
What about adding some logic to check if it is authored with a single digit number (5), your code corrects for that because we know nobody would actually set the delay for five one thousands of a second. 

On the flip side if it is authored with 5000 that would delay for five seconds.

Thoughts?
cc: @ryanmparrish @chrischrischris

Reply:
Thanks for the review @rgclayton Kyungwha and I discussed this as there is a block already in CC where we add time which is in milliseconds we agreed on having a common unit for time so that it's easier for authors. However I agree 5 seconds looks more readable, but we wanted to have parity across blocks to author time.

Reply:
To base an authoring pattern off of what's done in a consumer doesn't always work well with what goes into Milo.

With what I'm proposing, either option would work and I think it's a more friendly authoring pattern. Less questions/confusion about what values `delay` would take.

Reply:
I agree w @rgclayton - Using a simpler authoring seconds approach is likely preferred. 
The first thing I did testing this was hunt down what the delay value was. Seconds or milliseconds. 🤷  
I think the majority of authors would defer to simple seconds rather than mil. and having some logic that checks the length of the digit could be a nice way to have both.

Reply:
Thanks for the feedback @rgclayton and @ryanmparrish . I had a meeting with Ryan C and Kyung on Thursday to revisit the authoring pattern and I will get Ryan's suggestion implemented in this PR